pkg/skaffold/test/custom/custom.go

```diff
@@ -60,17 +57,17 @@ func New(cfg docker.Config, imageName string, wd string, ct latest.CustomTest) (
 }
 
 // Test is the entrypoint for running custom tests
-func (ct *Runner) Test(ctx context.Context, out io.Writer, artifacts []build.Artifact) error {
+func (ct *Runner) Test(ctx context.Context, out io.Writer, imageTag string) error {
 	event.TestInProgress()
-	if err := ct.runCustomTest(ctx, out, artifacts); err != nil {
+	if err := ct.runCustomTest(ctx, out, imageTag); err != nil {
```

Comment:
Maybe needs a follow up later or /cc @marlon-gamez to fix this in V2.
Add the artifact name for the test in progress event.